File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyStringRowHandleTreeViewElementsRow.cpp

#include "SelectRowCommand_ViewModelMyStringRowHandleTreeViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewModelMyStringRowHandleTreeViewElementsRow::getColumnCellLabelText( ) 
  {
    return this->columnCellLabelText;
  }
  std::string SelectRowCommand_ViewModelMyStringRowHandleTreeViewElementsRow::getRowHandle( ) 
  {
    return this->RowHandle;
  }
  int SelectRowCommand_ViewModelMyStringRowHandleTreeViewElementsRow::getRowDepth( ) 
  {
    return this->RowDepth;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyStringRowHandleTreeViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewModelMyStringRowHandleTreeViewElementsRow.cs
/// </filename>
namespace commands
{
    public class SelectRowCommand_ViewModelMyStringRowHandleTreeViewElementsRow
    {
        private string columnCellLabelText;

        public virtual string getColumnCellLabelText()
        {
            return this.columnCellLabelText;
        }

        private string RowHandle;

        public virtual string getRowHandle()
        {
            return this.RowHandle;
        }

        private int RowDepth;

        public virtual int getRowDepth()
        {
            return this.RowDepth;
        }
    }
}
